doc/installing.rst

```diff
@@ -24,84 +24,93 @@ KWallet, so that the sync client can login automatically.
 Improvements and New Features
 -----------------------------
 
-The 1.8 client has a number of enhancements and performance improvements, 
-including:
-
-* Graphical selective sync file and folder selector. Select which files and 
-  folders to sync with just a few clicks.
-* Selective sync on account setup. Set up which files and folders you want to 
-  sync when you create a new ownCloud account.
-* File manager integration: share files from Windows Explorer, Mac OS X Finder, 
-  and Nautilus on Linux.
-* Faster uploads and downloads.
-
-.. note:: When you upgrade from 1.7, restart Windows to ensure that all new 
+The 2.0 release of the ownCloud desktop sync client has many new features and 
+improvements.
+
+  * Multi-account support
+  * Many UI improvements
+  * Accessibility improvements (high contrast schemes)
+  * Automatic Bandwidth Throttling
+  * Don't show redundant directory entries in activity log
+  * Remove deleted accounts properly from toolbar
+  * File manager integration: Show hidden files as ignored
+  * Ignoring hidden files: Fix glitches when migrating from 1.8
+  * Show wizard when last account was deleted
```

Comment:
I would remove the fife entries before as they are rather bug fixes than new features.